Mergin/validation.py

```diff
@@ -463,13 +463,13 @@ def warning_display_string(warning_id, url=None):
     elif warning_id == Warning.ATTACHMENT_WRONG_EXPRESSION:
         return "Expression for the default path in the attachment widget configuration might be wrong. <a href='{help_mgr.howto_attachment_widget()}'>Read more.</a>"
     elif warning_id == Warning.QGIS_SNAPPING_NOT_ENABLED:
-        return "Snapping is currently disabled in this QGIS project, it will be thus disabled in Mergin Maps Input"
+        return "Snapping is currently disabled in this QGIS project, it will be thus disabled in Mergin Maps mobile app"
     elif warning_id == Warning.MERGIN_SNAPPING_NOT_ENABLED:
-        return "Snapping is currently enabled in this QGIS project, but not enabled in Mergin Maps Input"
+        return "Snapping is currently enabled in this QGIS project, but not enabled in Mergin Maps mobile app"
     elif warning_id == Warning.MISSING_DATUM_SHIFT_GRID:
         return "Required datum shift grid is missing, reprojection may not work correctly. <a href='fix_datum_shift_grids'>Fix the issue.</a>"
     elif warning_id == Warning.SVG_NOT_EMBEDDED:
-        return "SVGs used for layer styling are not embedded in the project file, as a result those symbols won't be displayed in Mergin Maps Input"
+        return "SVGs used for layer styling are not embedded in the project file, as a result those symbols won't be displayed in Mergin Maps mobile app"
```

Comment:
Let's stick with the short version here
```suggestion
        return "SVGs used for layer styling are not embedded in the project file, as a result those symbols won't be displayed in the mobile app"
```

Reply:
Resolved in latest commit, favored the shorter version except in the case of repetition